### ISSUE TITLE
[BUG] Script exception: Maps may only have keys of type string, int or an enumerated type.

### ISSUE THREAD

**Issue by Crotha** [loathers/garbage-collector#1801]
**Describe the bug**
After an Standard Ascension (via Autoscend), I ran garbo. Garbo started, encountered this (this may not be part of the problem, I don't know) :
```
Page (236 of 266)...
Unknown item found: tiny plastic Armory (11381, 897987307)
--------------------
11381 tiny plastic Elf Guard Armory 897987307 tp23_armory.gif accessory t 0
tiny plastic Elf Guard Armory 0 none
Item tiny plastic Elf Guard Armory Weapon Damage: +3
--------------------
Page (237 of 266)...
```

and then after updating mallprices.txt:

```
Page (266 of 266)...
Search complete.
Updating mallprices.txt with 7893 prices.
Script exception: Maps may only have keys of type string, int or an enumerated type.
```

Restarting the script now results in this:

```
> garbo ascend

Using adventure value 6500.
Local Version: aa9414896ddd51ced78f890f46a1a9f3e8112c4a (built from main@210e04a9e7c5c9ecfff35af4ae9b5f775bfb34a6)
Garbo is up to date!
Script exception: Maps may only have keys of type string, int or an enumerated type.
```

Reinstalling the script just leads to the same error directly.    
Deleting mallprices.txt does nothing, so I assume that has nothing to do with it.

**Comment by Crotha** [comment]
Short addendum:
garbo help works, but garbo simdiet doesn't:

```
> garbo simdiet

Using adventure value 6500.
Local Version: aa9414896ddd51ced78f890f46a1a9f3e8112c4a (built from main@210e04a9e7c5c9ecfff35af4ae9b5f775bfb34a6)
Garbo is up to date!
Script exception: Maps may only have keys of type string, int or an enumerated type.
```

**Comment by Crotha** [comment]
I figured out what the problem (?) was:    
For some reason, weird stuff was in my maximizerMRUList, seemingly. I only noticed because when I wanted to do some manual stuff, my Maximizer had no longer any strings in it saved (in the dropdown) and wasn't able to save new strings in there -> looked at the prefref, it was... huge. 
set the value to {} first, maximizer still didn't work, deleted the preferences from the setting file completely, restarted mafia, maximizer had a faulty string (deleted via the preference-menu of mafia), added a new string (just to see if it works), maximizer was normal again.    

And now garbo works.
The above might seem like I was just doing things by poking it until it works - and that's exactly what I did, so nothing was really coherent.    
The only thing I can actually say is: 
- Maximizer was in some way fucked up, no idea how - as only autoscend run before, maybe they did it? 
- So one of the maximizer-prefrefs having a wrong key seemingly prevented garbo from running.

Sorry for the write-up here, turns out garbo once again did nothing wrong and I wrongly blamed it!
